tracer/src/Datadog.Trace.ClrProfiler.Native/rejit_preprocessor.cpp

```diff
@@ -65,38 +64,43 @@ void RejitPreprocessor<RejitRequestDefinition>::ProcessTypeDefForRejit(const Rej
             continue;
         }
 
-        // Compare if the current mdMethodDef contains the same number of arguments as the
-        // instrumentation target
         const auto numOfArgs = functionInfo.method_signature.NumberOfArguments();
```

Comment:
This is already declared inside the `is_exact_signature_match` and is not required here.

Reply:
Fixed in c2ce735462580e6ff1d8ca6e1c92df5ab231afdc